common/test/common/JsonUtils.java

```diff
@@ -0,0 +1,19 @@
+package common;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+public class JsonUtils {
+
+    public static JsonNode readJsonNodeFromResource(final String resourcePath) throws IOException {
+        final ObjectMapper mapper = new ObjectMapper();
```

Comment:
I would not create per method call the Object mapper. Use a static member and nether give that mapper to the outside world.